<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToDoListExport</class>
 <widget class="QDialog" name="ToDoListExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export ToDo List</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTreeWidget" name="exportList">
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Type</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="recipientFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Recipients:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="recipientList"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportButton">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ToDoListExport</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exportButton</sender>
   <signal>clicked()</signal>
   <receiver>ToDoListExport</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>699</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
